pkg/labels/labels.go

```diff
@@ -269,6 +267,7 @@ func (l *Label) UnmarshalJSON(data []byte) error {
 		Value  string `json:"value,omitempty"`
 	}
 
+	decoder := json.NewDecoder(bytes.NewReader(data))
 	err := decoder.Decode(&aux)
```

Comment:
I think we can avoid using a decoder and directly attempt to unmarshal, no?
```suggestion
	err := json.Unmarshal(data, &aux)
```
Same comment applies for the other instance a few lines below.

Reply:
Good point, I'll fold that change into the commit touching `pkg/labels`.